places-solution-integrations/analytics/attach-data.md

```diff
@@ -0,0 +1,70 @@
+# Attaching Places Data to Analytics Events via Launch Rules
+
+This document assumes that you have Adobe Places implemented in your application. If you need help implementing Adobe Places, see [Places Extensions for AEP SDK](../../configure-places-in-the-sdk/README.md).
```

Comment:
Please reword the start of the 2nd sentence like this:
"For more information about implementing Places, see..."